### ISSUE TITLE
Clarification on Concurrent Mode Support

### ISSUE THREAD

**Issue by samcooke98** [mobxjs/mobx-react-lite#234]
On the docs, specifically this page https://mobx-react.js.org/state-local there is a commen:

>Note that using a local store might conflict with future React features like concurrent rendering.

Is it possible to address _why_ this might conflict? What does conflict mean? 

I've seen some discussion about React's ability to rebase, and Michel Westrate suggesting that mobx shouldn't be used for local component state - https://github.com/facebook/react/issues/15317#issuecomment-481727125 - Is this still a valid claim still?

**Comment by FredyC** [comment]
I guess somebody should actually try it out now when Concurrent is getting finalized. We have some theoretical solution in the `next` branch which I wanted to release as 2.0 soon, but given that Concurrent is out, it needs to be thoroughly tested.

**Comment by zenVentzi** [comment]
I tried reading through some of the discussions on twitter and other github issues but I couldn't make sense of the whole situation.

In layman terms, how should I approach the situation with local store if I intend on using React's Concurrent mode? Some clarity will be much appreciated!

**Comment by FredyC** [comment]
I am unsure about Concurrent mode gotchas myself. First of all, we need to try to run current tests in Concurrent mode. It should be possible with [RTL custom render](https://testing-library.com/docs/react-testing-library/setup#custom-render). Help appreciated if anyone is willing to give it a try.

**Comment by FredyC** [comment]
Just referencing an issue 274 that might be worth investigating with Concurrent mode and perhaps add into tests.
